src/app/utils/field-validation/validators/emailValidator.ts

```diff
@@ -38,7 +38,9 @@ const makeEmailDomainValidator: EmailValidatorConstructor =
     const emailAddress = String(answer)
     if (!(isVerifiable && hasAllowedEmailDomains && allowedEmailDomains.length))
       return right(response)
-    const emailDomain = ('@' + emailAddress.split('@').pop()).toLowerCase()
+    const emailDomain = ('@' + emailAddress.split('@').pop())
+      .toLowerCase()
+      .trim()
```

Comment:
This is not the error we were seeing though is it? Don't we need to trim before the `isEmail() check line 24 too?

Reply:
![Uploading image.png…]()

Reply:
Just for extra info, the domain error is:
`EmailValidator:\t answer is not a valid email domain`
while the plain email error is
`EmailValidator:\t answer is not a valid email`

![image](https://user-images.githubusercontent.com/935223/208078117-e151f337-0515-40ea-8793-8a6ec12b0a8d.png)

Reply:
dang, you are right sorry my bad

Reply:
rewrote history, fixed in [9662591](https://github.com/opengovsg/FormSG/pull/5581/commits/96625918646779c5575aad563ffd521fd44776bc)